### PR TITLE
Updated to meet BPs

### PR DESCRIPTION
**Pull request by bmoore-msft**
### Description of the change
Updated the sample to adhere to best practices

### CODE REVIEW

File: 101-vm-simple-windows/azuredeploy.parameters.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "contentVersion": "1.0.0.0",
   "parameters": {
     "adminUsername": {
-      "value": "admin123"
+      "value": "ghuser"
```
Comment (philon-msft) [inline 101-vm-simple-windows/azuredeploy.parameters.json]: For adminUsername value, I'd use "GEN-USER"

File: 101-vm-simple-windows/azuredeploy.json [hunk 2]
```diff
@@ -26,10 +26,13 @@
...
+        "2016-Nano-Server-Technical-Preview",
+        "2016-Technical-Preview-with-Containers",
+        "Windows-Server-Technical-Preview"
       ],
       "metadata": {
-        "description": "The Windows version for the VM. This will pick a fully patched image of this given Windows version. Allowed values: 2008-R2-SP1, 2012-Datacenter, 2012-R2-Datacenter."
+        "description": "The Windows version for the VM. This will pick a fully patched image of this given Windows version."
       }
     }
   },
```
Comment (philon-msft) [inline 101-vm-simple-windows/azuredeploy.json]: Putting the storageAccountType in a variable means the user won't get any suggestions in the editor about what different types are available. Should we eliminate variables like these?

Comment (philon-msft) [inline 101-vm-simple-windows/azuredeploy.json]: Same for publicIPAddressType, vmSize, imagePublisher, imageOffer

Comment (philon-msft) [inline 101-vm-simple-windows/azuredeploy.json]: Do we need OSDiskName and dataDisk1VhdName variables? Seems unlikely that users will want to change them, and they add unnecessary indirection.

File: 101-vm-simple-windows/README.md [hunk 3]
```diff
@@ -7,4 +7,4 @@
     <img src="http://armviz.io/visualizebutton.png"/>
 </a>
 
-This template allows you to deploy a simple Windows VM using a few different options for the Windows version, using the latest patched version. This will deploy in West US on a D1 VM Size.
+This template allows you to deploy a simple Windows VM using a few different options for the Windows version, using the latest patched version. This will deploy a D1 size VM in the resource group location and return the FQDN of the VM.
```
Comment (bmoore-msft) [inline 101-vm-simple-windows/README.md]: I think this is ok, it follows the property name

File: 101-vm-simple-windows/azuredeploy.json (hunk 2 above)
Comment (bmoore-msft) [inline 101-vm-simple-windows/azuredeploy.json]: yep

Comment (bmoore-msft) [inline 101-vm-simple-windows/azuredeploy.json]: yep

File: 101-vm-simple-windows/README.md (hunk 3 above)
Comment (bmoore-msft) [inline 101-vm-simple-windows/README.md]: ok

File: 101-vm-simple-windows/azuredeploy.parameters.json (hunk 1 above)
Comment (bmoore-msft) [inline 101-vm-simple-windows/azuredeploy.parameters.json]: done

**Pull request merged**